test/e2e/specs/editor/various/preview.spec.js

```diff
@@ -184,6 +184,20 @@ test.describe( 'Preview', () => {
 		// FIXME: The confirmation dialog is not named yet.
 		await page.click( 'role=dialog >> role=button[name="OK"i]' );
 
+		// Wait for the status change.
```

Comment:
Let's add this PR to the comment so that it's clearer why we need to explicitly wait for it?